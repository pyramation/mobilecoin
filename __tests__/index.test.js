import mobilecoin from '../src';
import { client } from '../src';

it('mobilecoin', () => {
  expect(mobilecoin).toMatchSnapshot();
});
it('client', () => {
  expect(client).toMatchSnapshot();
});
