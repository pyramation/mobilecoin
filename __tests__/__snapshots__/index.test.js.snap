// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`client 1`] = `[Function]`;

exports[`mobilecoin 1`] = `
Object {
  "assignAddressForAccount": [Function],
  "buildGiftCode": [Function],
  "buildTransaction": [Function],
  "checkB58PaymentRequest": [Function],
  "checkGiftCodeStatus": [Function],
  "claimGiftCode": [Function],
  "createAccount": [Function],
  "createReceiverReceipts": [Function],
  "exportAccountSecrets": [Function],
  "getAccount": [Function],
  "getAccountStatus": [Function],
  "getAllAccounts": [Function],
  "getAllAddressesForAccount": [Function],
  "getAllGiftCodes": [Function],
  "getAllTransactionLogsForAccount": [Function],
  "getAllTxosForAccount": [Function],
  "getBalanceForAccount": [Function],
  "getConfirmations": [Function],
  "getGiftCode": [Function],
  "getNetworkStatus": [Function],
  "getTransactionLog": [Function],
  "getTxo": [Function],
  "getWalletStatus": [Function],
  "importAccount": [Function],
  "importLegacyAccount": [Function],
  "removeAccount": [Function],
  "removeGiftCode": [Function],
  "submitGiftCode": [Function],
  "submitTransaction": [Function],
  "updateAccountName": [Function],
  "validateConfirmation": [Function],
}
`;

exports[`works 1`] = `
Object {
  "assignAddressForAccount": [Function],
  "buildGiftCode": [Function],
  "buildTransaction": [Function],
  "checkB58PaymentRequest": [Function],
  "checkGiftCodeStatus": [Function],
  "claimGiftCode": [Function],
  "createAccount": [Function],
  "createReceiverReceipts": [Function],
  "exportAccountSecrets": [Function],
  "getAccount": [Function],
  "getAccountStatus": [Function],
  "getAllAccounts": [Function],
  "getAllAddressesForAccount": [Function],
  "getAllGiftCodes": [Function],
  "getAllTransactionLogsForAccount": [Function],
  "getAllTxosForAccount": [Function],
  "getBalanceForAccount": [Function],
  "getConfirmations": [Function],
  "getGiftCode": [Function],
  "getNetworkStatus": [Function],
  "getTransactionLog": [Function],
  "getTxo": [Function],
  "getWalletStatus": [Function],
  "importAccount": [Function],
  "importLegacyAccount": [Function],
  "removeAccount": [Function],
  "removeGiftCode": [Function],
  "submitGiftCode": [Function],
  "submitTransaction": [Function],
  "updateAccountName": [Function],
  "validateConfirmation": [Function],
}
`;
